guacamole/src/main/webapp/app/client/controllers/clientController.js

```diff
@@ -522,21 +539,44 @@ angular.module('client').controller('clientController', ['$scope', '$routeParams
             }
         }
 
+        // If only Ctrl-Alt-End is pressed, and we have a one keysym from each
+        // group, and one key is being released, send Ctrl-Alt-Delete.
+        if (END_KEYS[keysym] &&
+            !_.isEmpty(_.pick(ALT_KEYS, currentKeysPressedKeys)) &&
+            !_.isEmpty(_.pick(CTRL_KEYS, currentKeysPressedKeys))
+        ) {
+                event.preventDefault();
+                delete keysCurrentlyPressed[keysym];
+                substituteKeysPressed[keysym] = DEL_KEY;
+                $rootScope.$broadcast('guacSyntheticKeydown', DEL_KEY);
+        }
+
     });
 
     // Update pressed keys as they are released, synchronizing the clipboard
     // with any data that appears to have come from those key presses
     $scope.$on('guacKeyup', function keyupListener(event, keysym, keyboard) {
 
+        var currentKeysPressedKeys = Object.keys(keysCurrentlyPressed);
```

Comment:
This variable doesn't seem to be used here.

Reply:
Oops, removed.